Documentation/usage/dlv.md

```diff
@@ -41,10 +41,9 @@ Pass flags to the program you are debugging using `--`, for example:
 * [dlv core](dlv_core.md)	 - Examine a core dump.
 * [dlv debug](dlv_debug.md)	 - Compile and begin debugging main package in current directory, or the package specified.
 * [dlv exec](dlv_exec.md)	 - Execute a precompiled binary, and begin a debug session.
-* [dlv replay](dlv_replay.md)	 - Replays a rr trace.
```

Comment:
I am sorry reply so later.
```go run ./scripts/gen-usage-docs.go``` automatically deletes this command, do I need to manually add it back?